assets/js/vue/components/information-product/InformationProductForm.vue

```diff
@@ -190,7 +193,17 @@ export default {
       addedRgShortName: '',
     };
   },
-
+  computed: {
+    researchGroupsSelected () {
+      return this.form.selectedResearchGroups.length > 0;
```

Comment:
I think this is too late here. We want to return whether or not there is a value for the add-research-groups field.